source/includes/inkling-reference/_algorithms.html.md

```diff
@@ -0,0 +1,196 @@
+| `conv_limit_dim` => string (`conv_limit_dim => “x”`) | A string that represents which dimension of the input shape to be used as the convolutional filter sizes. Strings supported: “max”, “min”, “x”, “y”, “0”, “1”. By default, it’s `None`, which uses the original input shapes as the filter sizes. “max” uses the maximum number of x and y inputs as the filter sizes. “min” uses the minimum number of x and y inputs as the filter sizes. “x” or “0” uses the x dimension input shape as the filter sizes. “y” or “1” uses the y dimension input shape as the filter sizes. (The example uses the input x dimension shape as the convolutional filter sizes.) |
+
+
+### DQN-Specific Parameters
+
+In addition to the first (shared) table of parameters, DQN also has four additional parameters available.
+
+| Parameter (Example use)      | Description |
+| -                            | -           |
+| `exploration_decay` => float <br> (`exploration_decay => 0.00001`) | The exploration decay in DQN with the data type float. |
+| `samples_per_train` => int <br> (`samples_per_train => 10`) | The ratio of number of data sampling and training. |
+| `q_learning_rate` => float <br> (`q_learning_rate => 0.0001`) | The learning rate for training the Q network. |
+| `gamma` => float <br> (`gamma => 0.99`) | The discount rate for the Bellman Equation. |
+
+### DDPG-Specific (and DiscreteDDPG) Parameters
+
+In addition to the first (shared) table of parameters, DDPR (and DiscreteDDPG) also has three additional parameters available.
+
+| Parameter (Example use)      | Description |
+| -                            | -           |
+| `mu_learning_rate` => float <br> (`mu_learning_rate => 0.0001`) | The learning rate for training the Mu network. |
+| `q_learning_rate` => float <br> (`q_learning_rate => 0.0001`) | The learning rate for training the Q network. |
+| `gamma` => float <br> (`gamma => 0.99`) | The discount rate for the Bellman Equation. |
+
+### TRPO-Specific Parameters
+
+In addition to the first (shared) table of parameters, TRPO also has one additional parameter available.
+
+| Parameter (Example use)      | Description |
+| -                            | -           |
+| `samples_per_train` => int <br> (`samples_per_train => 10`) | The ratio of number of data sampling and training. |
+
+### Q-Table Parameters
+
+These are the only three parameters available for Q-Tables. They do not share any of the parameters from the first (shared) table.
+
+| Parameter (Example use)      | Description |
+| -                            | -           |
+| `exploration_decay` => float <br> (`exploration_decay => 0.00001`) | The exploration decay in DQN with the data type float. |
```

Comment:
"DQN". 

Also, here and a few other places: the type is specified on the left, so no need to repeat in the description.